### PR TITLE
chore: add volunteer NilsJacobsen

### PR DESCRIPTION
**Pull request by NilsJacobsen**
Hey folks, awesome idea. I'm in. 🚀

### CODE REVIEW

File: data/users/NilsJacobsen.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Nils Jacobsen",
+  "bio": "Frontend Dev and Interaction Designer",
+  "image": "https://pbs.twimg.com/profile_images/1491468272226779145/P4zfulvL_400x400.jpg",
```
Comment (codewithvoid) [inline data/users/NilsJacobsen.json:4]: use github image url instead? https://avatars.githubusercontent.com/u/58360188?v=4

File: data/users/NilsJacobsen.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Nils Jacobsen",
+  "bio": "Frontend Dev and Interaction Designer",
```
Comment (codewithvoid) [inline data/users/NilsJacobsen.json:3]: Pick 1 title here. Preferably Frontend Dev as we don't have designer related topics atm.

File: data/users/NilsJacobsen.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  "image": "https://pbs.twimg.com/profile_images/1491468272226779145/P4zfulvL_400x400.jpg",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["react", "nextjs", "recoil", "tailwind"]
+    },
+    {
+      "slug": "interaction-design",
```
Comment (codewithvoid) [inline data/users/NilsJacobsen.json:11]: Same as above. Categories are predefined [here](https://github.com/codewithvoid/AskDevs/blob/main/public/categories.json)

File: data/users/NilsJacobsen.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Nils Jacobsen",
+  "bio": "Frontend Dev and Interaction Designer",
+  "image": "https://pbs.twimg.com/profile_images/1491468272226779145/P4zfulvL_400x400.jpg",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["react", "nextjs", "recoil", "tailwind"]
```
Comment (codewithvoid) [inline data/users/NilsJacobsen.json:8]: keep topics to max 2-3

**Comment by codewithvoid** [comment]
Thanks for volunteering @NilsJacobsen. Added some comments PTAL.

**Review by codewithvoid — CHANGES_REQUESTED** [review]

**Comment by NilsJacobsen** [comment]
@codewithvoid I applied your change requests.

**Review by codewithvoid — APPROVED** [review]

**Comment by codewithvoid** [comment]
Perfect. Welcome Nils!

**Pull request merged**